### PR TITLE
[WKP-848] Update weave-net IPALLOW_RANGE on init

### PR DESCRIPTION
**Pull request by foot**
- If you are gitopsing weave-net

### CODE REVIEW

File: cmd/wksctl/init/init.go [hunk 1]
```diff
@@ -181,10 +217,10 @@ func updateManifests(options initOptionType) error {
 			}
 			return nil
 		})
-	if matches < len(updates) {
+	if matches < 2 {
 		return errors.New("Both 'flux.yaml' and 'wks-controller.yaml' must be present in the repository")
```
Comment (foot) [inline cmd/wksctl/init/init.go:221]: Need a better condition to show this error message now. as `weave-net.yaml` is optional and would make 3.

File: cmd/wksctl/init/init.go [hunk 2]
```diff
@@ -38,13 +45,12 @@ type manifestUpdate struct {
 var (
 	// Cmd represents the init command
 	Cmd = &cobra.Command{
-		Use:           "init",
-		Short:         "Update stored kubernetes manifests to match the local cluster environment",
-		Long:          "'wksctl init' configures existing kubernetes 'flux.yaml' and 'wks-controller.yaml' manifests in a repository with information about the local GitOps repository, the preferred weave system namespace, and current container image tags. The files can be anywhere in the repository. If either file is absent, 'wksctl init' will return an error.",
-		Example:       "wksctl init --namespace=wksctl --git-url=git@github.com:haskellcurry/lambda.git --git-branch=development --git-path=src",
-		RunE:          initRun,
-		SilenceErrors: true,
```
Comment (foot) [inline cmd/wksctl/init/init.go:46]: Not sure why this was Silencing errors

File: cmd/wksctl/init/init.go [hunk 3]
```diff
@@ -144,6 +153,32 @@ func updateControllerManifests(contents []byte, options initOptionType) ([]byte,
 	return withVersion, nil
 }
 
+func updateWeaveNetManifests(contents []byte, options initOptionType) ([]byte, error) {
+	clusterManifestPath := (path.Join(options.localRepoDirectory, options.clusterManifestPath))
+	machinesManifestPath := (path.Join(options.localRepoDirectory, options.machinesManifestPath))
```
Comment (foot) [inline cmd/wksctl/init/init.go:158]: Should maybe take into account gitPath? Not sure..

File: cmd/wksctl/init/init.go [hunk 4]
```diff
@@ -144,6 +153,32 @@ func updateControllerManifests(contents []byte, options initOptionType) ([]byte,
...
+	log.Debug("Loaded spec")
+
+	podsCIDRBlocks := sp.Cluster.Spec.ClusterNetwork.Pods.CIDRBlocks
+	if len(podsCIDRBlocks) > 0 && podsCIDRBlocks[0] != "" {
+		// setting the pod CIDR block is currently only supported for the weave-net CNI
+		log.Debug("Updating manifest..")
+		manifests, err := wksos.SetWeaveNetPodCIDRBlock([][]byte{contents}, podsCIDRBlocks[0])
+		for i, m := range manifests {
+			log.Debugf("Manifests! %v: %v", i, strings.Split(string(m), "\n")[:2])
+		}
```
Comment (foot) [inline cmd/wksctl/init/init.go:170]: Remove debugging code

File: cmd/wksctl/init/init.go [hunk 5]
```diff
@@ -19,32 +24,34 @@ import (
...
-		Use:           "init",
-		Short:         "Update stored kubernetes manifests to match the local cluster environment",
-		Long:          "'wksctl init' configures existing kubernetes 'flux.yaml' and 'wks-controller.yaml' manifests in a repository with information about the local GitOps repository, the preferred weave system namespace, and current container image tags. The files can be anywhere in the repository. If either file is absent, 'wksctl init' will return an error.",
-		Example:       "wksctl init --namespace=wksctl --git-url=git@github.com:haskellcurry/lambda.git --git-branch=development --git-path=src",
-		RunE:          initRun,
-		SilenceErrors: true,
-		SilenceUsage:  true,
+		Use:          "init",
+		Short:        "Update stored kubernetes manifests to match the local cluster environment",
+		Long:         "'wksctl init' configures existing kubernetes 'flux.yaml' and 'wks-controller.yaml' manifests in a repository with information about the local GitOps repository, the preferred weave system namespace, and current container image tags. The files can be anywhere in the repository. If either file is absent, 'wksctl init' will return an error.",
```
Comment (dinosk) [inline cmd/wksctl/init/init.go:51]: and 'weave-net.yaml'

**Pull request merged**